### PR TITLE
Materialize

### PR DESCRIPTION
**Pull request by gshigeto**
Added Materialize to our project for #1

### CODE REVIEW

**Comment by jbw91** [comment]
@gshigeto You added a `.less` file, but I can't see anywhere you've actually installed Less. Therefore when I serve it, it can't find the css file that's being referenced, because there's no compilation of the less. Did you maybe forget to use `--save-dev` while installing it or something?

File: src/system-config.ts [hunk 1]
```diff
@@ -7,10 +7,16 @@
  **********************************************************************************************/
 /** Map relative paths to URLs. */
 const map: any = {
+	"materialize": "vendor/materialize-css",
```
Comment (jbw91) [inline src/system-config.ts]: @gshigeto All these quotes should be single quotes. (Do you have the tslint package installed?)

**Comment by gshigeto** [comment]
@jbw91 Are you talking about the main.less? Is it not referenced by the main.css by adding it in the index?

File: src/system-config.ts (hunk 1 above)
Comment (gshigeto) [inline src/system-config.ts]: Maybe I haven't enabled the package or have is installed globally?

**Comment by jbw91** [comment]
It's referencing main.css, but since we have no less compilation happening, that less file is never actually turned into a css file.

**Comment by drewctate** [comment]
Looks great!

**Pull request merged**